docs/tracing_ui.md

```diff
@@ -80,7 +80,7 @@ In addition to the general layer properties mentioned above, `color` and `segmen
 ![Video: Visualize Machine-Learning Predictions](https://www.youtube.com/watch?v=JZLKB9GfMgQ)
 
 #### Skeleton Annotation Layer
-The skeleton annotation layer contains any skeletons/trees that you add to your dataset. You can quickly toggle the visibility of all skeletons by enabling/disabling this layer.
+The skeleton annotation layer contains any trees that you add to your dataset. You can quickly toggle the visibility of all trees by enabling/disabling this layer.
```

Comment:
> As far as I know, using "trees" instead of "skeletons" was what we agreed upon in the meeting about https://github.com/scalableminds/webknossos/issues/6053.

But the sentence should probably read as: "You can quickly toggle the visibility of all skeleton annotations by enabling/disabling this layer." At least, that we it's previous meaning. I understand the term `skeletons` as the shorthand for the plural `skeleton annotations` here.

Reply:
Hm, admittedly the terminology is hard, but I don't really agree here. We tell the user that there is "a (one) skeleton annotation layer" that "contains trees". Toggling the layer will change the visibility of these trees. Your suggestion would say that "one skeleton annotation layer" contains "multiple skeleton annotations". The rest of the UI uses "trees" for that, so I'd stick to that here. Linking to the nodes-and-trees section (from my comment above) should ensure that the user understands what exactly the skeleton annotation layer contains.

Reply:
I get your point, but I still it confusing. Any user will say "I traced the skeletons for 10 neurons. Now I have 10 skeleton reconstructions." not "I traced the 10 trees for 10 neurons. Now I have the 10 tree reconstructions." ;-)

Reply:
according to our terminology they should say "I created 10 trees for 10 neurons. Now I have 10 trees", but I understand that this might be unlikely :stuck_out_tongue: I'm fine with the newest change @Dagobert42 pushed :+1:

Reply:
A skeleton, a tree and an annotation walk into a bar. The user disables something in the tab, now they're all gone.